### ISSUE TITLE
With RBAC enabled, trigger get gives 500:Internal server error 

### ISSUE THREAD

**Issue by humblearner** [StackStorm/st2#3348]
```
$ st2 trigger get core.st2.webhook
Traceback (most recent call last):
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/commands/resource.py", line 162, in get_resource_by_pk
    instance = self.manager.get_by_id(pk, **kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/models/core.py", line 39, in decorate
    return func(*args, **kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/models/core.py", line 204, in get_by_id
    self.handle_error(response)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2client/models/core.py", line 165, in handle_error
    response.raise_for_status()
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/requests/models.py", line 862, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
HTTPError: 500 Server Error: Internal Server Error
MESSAGE: Internal Server Error for url: http://127.0.0.1:9101/v1/triggertypes/core.st2.webhook
Trigger "core.st2.webhook" is not found.
```

st2api.log:
--------------

```
2017-04-12 00:33:57,775 140260897577712 INFO logging [-] 36d75dfb-c2ee-46d0-a375-50f2eba40702 - GET /v1/triggertypes/core.st2.webhook with query={} (remote_addr='127.0.0.1',method='GET',request_id='36d75dfb-c2ee-46d0-a375-50f2eba40702',query={},path='/v1/triggertypes/core.st2.webhook')
2017-04-12 00:33:57,777 140260897577712 AUDIT auth [-] Token with id "58ed5ff055fc8c1ec02d788d" is validated.
2017-04-12 00:33:57,780 140260897577712 ERROR error_handling [-] API call failed: 'NoneType' object has no attribute 'split'
Traceback (most recent call last):
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/middleware/error_handling.py", line 39, in __call__
    return self.app(environ, start_response)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/middleware/logging.py", line 69, in __call__
    retval = self.app(environ, custom_start_response)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/middleware/cors.py", line 75, in __call__
    return self.app(environ, custom_start_response)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/router.py", line 429, in as_wsgi
    resp = self(req)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/router.py", line 402, in __call__
    resp = func(**kw)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2api/controllers/v1/triggers.py", line 59, in get_one
    return super(TriggerTypeController, self).get_one(triggertype_ref_or_id, **kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2api/controllers/resource.py", line 360, in get_one
    return self._get_one(ref_or_id, from_model_kwargs=from_model_kwargs)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2api/controllers/resource.py", line 379, in _get_one
    permission_type=permission_type)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/rbac/utils.py", line 119, in assert_user_has_resource_db_permission
    permission_type=permission_type)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/rbac/utils.py", line 300, in user_has_resource_db_permission
    resolver = resolvers.get_resolver_for_permission_type(permission_type=permission_type)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/rbac/resolvers.py", line 867, in get_resolver_for_permission_type
    resource_type = PermissionType.get_resource_type(permission_type=permission_type)
  File "/opt/stackstorm/st2/local/lib/python2.7/site-packages/st2common/rbac/types.py", line 139, in get_resource_type
    split = permission_type.split('_')
AttributeError: 'NoneType' object has no attribute 'split' (_exception_data={},_exception_class='AttributeError',_exception_message="'NoneType' object has no attribute 'split'")
```

**Comment by Kami** [comment]
Thanks for reporting this.

It's possible a regression in new API changes - either @enykeev or I should have a look shortly.

**Comment by Kami** [comment]
Never mind, I see you already pushed a fix #3349 - thanks :)
